daffodil-io/src/main/scala/org/apache/daffodil/io/DirectOrBufferedDataOutputStream.scala

```diff
@@ -142,12 +144,28 @@ private[io] class ByteArrayOrFileOutputStreamWithGetBuf(
  * is not related to a layer, that means the associated Java OutputStream came
  * from the user and it is the users responsibility to close it. The isLayer
  * provides the flag to know which streams should be closed or not.
+ *
+ * chunkSizeInBytes is used when the buffered output stream is using a file as
+ * its buffer. This is the size of chunks that will be read into memory before
+ * being written out to a file.
```

Comment:
Should be "before being written to the direct output stream". The direct output stream might not also be a file